File: Codeforces/231A - Team/231A.cpp
#include <iostream>
using namespace std;

int main() {
    int n, cnt = 0, petya, vasya, tonya;
    cin >> n;
    for(int i = 0; i < n; i++) {
        cin >> petya >> vasya >> tonya;
        if(petya && vasya) cnt++;
        else if(vasya && tonya) cnt++;
        else if(petya && tonya) cnt++;
    }
    cout << cnt;
}

File: Codeforces/231A - Team/231A.py
n = int(input())
cnt = 0
for i in range(n):
    petya, vasya, tonya = map(int, input().split())
    if petya and vasya:
        cnt += 1
    elif vasya and tonya:
        cnt += 1
    elif petya and tonya:
        cnt += 1
print(cnt)